### PR TITLE
Delete Frcanalytics.com.xml

### PR DESCRIPTION
**Pull request by Folant**
Host does not exist

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
Can you update the reference in Digital-River.xml?

**Comment by Folant** [comment]
Done.

File: src/chrome/content/rules/Digital-River.xml [hunk 1]
```diff
@@ -76,25 +75,12 @@
...
-			<test url="http://info.digitalriver.com/BlueHornet-Demo.html" />
-			<test url="http://info.digitalriver.com/Contact-Us.html" />
-			<test url="http://info.digitalriver.com/ContactUsCN.html" />
-			<test url="http://info.digitalriver.com/ManyWaysToPay.html" />
-			<test url="http://info.digitalriver.com/MyCommerceContactUs.html" />
-
 			<!--	+ve:
 					-->
 			<test url="http://info.digitalriver.com/css/mktLPSupport.css" />
 			<test url="http://info.digitalriver.com/rs/digitalriver/images/mycommerce_CMYK.jpg" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Digital-River.xml]: Can you remove this two tests as well?

Comment (Folant) [inline src/chrome/content/rules/Digital-River.xml]: Done.

Comment (gloomy-ghost) [inline src/chrome/content/rules/Digital-River.xml]: Please delete the comment above the tests and then we are ready to go

**Pull request merged**